Remove unused reconstruct_scopes function

cccclai opened the pull request:
## Summary
This pr https://github.com/pytorch/pytorch/pull/46893 remove the only usage of `reconstruct_scopes` function. As a follow up, clean up the unused class.


## Test plan

```
python test/mobile/test_lite_script_module.py
/Users/chenlai/pytorch/cmake-build-debug/bin/test_jit --gtest_filter=LiteInterpreterTest.*:LiteInterpreterTest/*.*:*/LiteInterpreterTest.*/*:*/LiteInterpreterTest/*.* --gtest_color=no

```
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#48822 Remove unused reconstruct_scopes function**

Differential Revision: [D25325012](https://our.internmc.facebook.com/intern/diff/D25325012)

cccclai committed:
Remove unused reconstruct_scopes function

[ghstack-poisoned]

cccclai committed:
Update on "Remove unused reconstruct_scopes function"


Differential Revision: [D25325012](https://our.internmc.facebook.com/intern/diff/D25325012)

[ghstack-poisoned]

cccclai committed:
Update on "Remove unused reconstruct_scopes function"

## Summary
This pr https://github.com/pytorch/pytorch/pull/46893 remove the only usage of `reconstruct_scopes` function. As a follow up, clean up the unused class.


## Test plan

```
python test/mobile/test_lite_script_module.py
/Users/chenlai/pytorch/cmake-build-debug/bin/test_jit --gtest_filter=LiteInterpreterTest.*:LiteInterpreterTest/*.*:*/LiteInterpreterTest.*/*:*/LiteInterpreterTest/*.* --gtest_color=no

```

Differential Revision: [D25325012](https://our.internmc.facebook.com/intern/diff/D25325012)

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
![image](https://user-images.githubusercontent.com/4588111/101186435-efc0e800-3607-11eb-9c4e-6c633d334255.png)

cccclai committed:
Update on "Remove unused reconstruct_scopes function"

## Summary
This pr https://github.com/pytorch/pytorch/pull/46893 remove the only usage of `reconstruct_scopes` function. As a follow up, clean up the unused class.


## Test plan

```
python test/mobile/test_lite_script_module.py
/Users/chenlai/pytorch/cmake-build-debug/bin/test_jit --gtest_filter=LiteInterpreterTest.*:LiteInterpreterTest/*.*:*/LiteInterpreterTest.*/*:*/LiteInterpreterTest/*.* --gtest_color=no

```

Differential Revision: [D25325012](https://our.internmc.facebook.com/intern/diff/D25325012)

[ghstack-poisoned]

cccclai committed:
Update on "Remove unused reconstruct_scopes function"

## Summary
This pr https://github.com/pytorch/pytorch/pull/46893 remove the only usage of `reconstruct_scopes` function. As a follow up, clean up the unused class.


## Test plan

```
python test/mobile/test_lite_script_module.py
/Users/chenlai/pytorch/cmake-build-debug/bin/test_jit --gtest_filter=LiteInterpreterTest.*:LiteInterpreterTest/*.*:*/LiteInterpreterTest.*/*:*/LiteInterpreterTest/*.* --gtest_color=no

```

Differential Revision: [D25325012](https://our.internmc.facebook.com/intern/diff/D25325012)

[ghstack-poisoned]

cccclai committed:
Update on "Remove unused reconstruct_scopes function"

## Summary
This pr https://github.com/pytorch/pytorch/pull/46893 remove the only usage of `reconstruct_scopes` function. As a follow up, clean up the unused class.


## Test plan

```
python test/mobile/test_lite_script_module.py
/Users/chenlai/pytorch/cmake-build-debug/bin/test_jit --gtest_filter=LiteInterpreterTest.*:LiteInterpreterTest/*.*:*/LiteInterpreterTest.*/*:*/LiteInterpreterTest/*.* --gtest_color=no

```

Differential Revision: [D25325012](https://our.internmc.facebook.com/intern/diff/D25325012)

[ghstack-poisoned]
